pkg/trace/operation.go

```diff
@@ -122,6 +122,14 @@ func (o *Operation) Errorf(format string, args ...interface{}) {
 	Logger.Errorf("%s: %s", o.header(), fmt.Sprintf(format, args...))
 }
 
+func (o *Operation) Warnf(format string, args ...interface{}) {
+	Logger.Warnf("%s: %s", o.header(), fmt.Sprintf(format, args...))
+}
+
+func (o *Operation) SPrintf(format string, args ...interface{}) string {
```

Comment:
Not sure how I feel about this one.  Smells useful, but I think we can hide whatever we're after in a helper since ultimately the string we're building is suppose to end up in the log.